src/components/NewClusterModal.js

```diff
@@ -612,7 +612,7 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
         div({ style: styles.whiteBoxContainer }, [
           div({ style: { fontSize: '0.875rem', fontWeight: 600, marginBottom: '0.5rem' } }, ['Cloud compute configuration']),
           // TODO PD: remove the profile select
-          div({ style: { marginBottom: '1rem' } }, ['Select from one of the default runtime profiles or define your own']),
+          div({ style: { marginBottom: '1rem' } }, ['Select from one of the default cloud compute profiles or define your own']),
```

Comment:
I made this say `cloud compute` instead of `cloud environment` because it is talking specifically about the `Cloud compute configuration`